cluster/gce/gci/configure-helper.sh

```diff
@@ -55,6 +55,27 @@ function config-ip-firewall {
   if [[ "${ENABLE_METADATA_CONCEALMENT:-}" == "true" ]]; then
     iptables -A KUBE-METADATA-SERVER -j DROP
   fi
+
+  # Flush iptables nat table
+  iptables -t nat -F || true
```

Comment:
Curious why we flush this table.  Are we sure there is nothing else in the nat table when configure-helper.sh is called?
Its probably ok, but I would double check you aren't flushing something important out.

Reply:
We did this before. I just move the code up

Reply:
In https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/configure-vm.sh#L180, it looks like we're doing something similar.  The comment is, "Remove docker artifacts on minion nodes, if present."  Idk enough to know if that's similar here, but may be.  None of the other munging of `docker0` is present in this file, and it looks like maybe we just configure docker from the beginning to fix it (https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L839)?

I care about this to make https://github.com/kubernetes/kubernetes/pull/55360 work right (I need to configure that rule after the flush).